src/routes/api/requests.router.js

```diff
@@ -0,0 +1,15 @@
+import express from 'express';
+import auth from '../../middlewares/auth';
+import RequestsController from '../../controllers/requests.controller';
+import requestChecks from '../../middlewares/requests.middleware';
+
+const router = express.Router();
+
+/* Requests Routes Here */
+router.patch('/requests/:id',
```

Comment:
I think the agreed endpoint was `/request` without the 's'.

Reply:
Thanks for pointing this out. I will go ahead and resolve this

Reply:
Great.

Reply:
Again, great job.